scalableminds/webknossos
# Highlight commented nodes

~(Still waiting for final feedback from the lab.)~

### Mailable description of changes:
 - Commented nodes are now highlighted in the different viewports. Each commented node is painted in white and receives an additional border in the usual node color.

### Steps to test:
- Open a skeleton tracing
- Create, remove comments
- Reload page to see if comments are correctly rendered on initial render

### Test instance
https://highlightcommentednodes.webknossos.xyz

### Issues:
- fixes #2217 

------
